recipes-ni/ni-utils/files/udhcpd.wlan0.conf

```diff
@@ -0,0 +1,11 @@
+start 172.16.0.10
```

Comment:
Does it make sense to add this file to the busybox recipe in a udhcpd package instead, like the ifplugd.conf file [here](https://github.com/ni/meta-nilrt/blob/nilrt/master/dunfell/recipes-core/busybox/files/ifplugd.conf)?

Reply:
Yeah, I'll move it. I really don't like tangling more things into busybox - we've been slowly replacing busybox utilities with the full functionality stand alone ones but I guess for now that makes more sense.

Reply:
Closing this PR in favor of a new one against the busybox recipe.